specificatie/uris.yaml

```diff
@@ -0,0 +1,25 @@
+openapi: 3.0.3
+info:
+  title: Haal Centraal uri gerelateerde definities
+  version: 1.0.0
+paths: {}
+components:
+  headers:
+    Accept-Uri-Format:
+      description: |
+        Geeft aan welke uri format(en) de consumer kan verwerken
+      schema:
+        $ref: '#/components/schemas/UriFormatsEnum'
+    Content-Uri-Format:
+      description: |
+        Geeft aan welke uri format door de provider is toegepast
+      schema:
+        $ref: '#/components/schemas/UriFormatsEnum'
+  schemas:
+    UriFormatsEnum:
+      type:
```

Comment:
default toevoegen?

Reply:
Ik heb daar mijn twijfels over. Toevoegen van een default, zorgt ervoor dat gegenereerde code standaard deze default waarde invullen als er geen waarde is opgegeven.
Dit betekent dat de provider geen verschil meer ziet tussen 'er is geen waarde opgegeven' en 'de default is als waarde opgegeven'. Functioneel leiden beide tot hetzelfde resultaat, maar semantisch zijn ze niet gelijk.